shell/gpu/gpu_surface_delegate.h

```diff
@@ -2,14 +2,19 @@
 #define FLUTTER_SHELL_GPU_GPU_SURFACE_DELEGATE_H_
 
 #include "flutter/flow/embedded_views.h"
-#include "flutter/fml/macros.h"
 
 namespace flutter {
 
 class GPUSurfaceDelegate {
  public:
-  // Get a reference to the external views embedder. This happens on the same
-  // thread that the renderer is operating on.
+  virtual ~GPUSurfaceDelegate() {}
+
+  // Gets the view embedder that controls how the Flutter layer hierarchy split
+  // into multiple chunks should be composited back on-screen. This field is
+  // optional and the Flutter rasterizer will render into a single on-screen
+  // surface if this call returns a null external view embedder.
+  //
+  // This happens on the gpu thread.
   virtual ExternalViewEmbedder* GetExternalViewEmbedder() = 0;
```

Comment:
Minor nit: Can you keep the headerdoc format like in the header you moved this docstring from?

Reply:
+1